spec/collection-spec.coffee

```diff
@@ -1,18 +1,33 @@
-describe 'Brainstem.Collection', ->
-  collection = updateArray = null
+$ = require 'jquery'
+_ = require 'underscore'
+Backbone = require 'backbone'
```

Comment:
They are only shimmed for the browseified gem output to prevent including the dependencies in the gem bundle.